examples/rbd/fs-vol-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This is not useful at all for the users right as it's a replica `3` and pod can come upon any node? how about generating this template in e2e  tests only?

Reply:
For me, reading the template would be much clear to understand :-) Let's not complicate it? (Just give it an another thought in this direction). Maybe you can suggest to me a location that holds templates for e2e tests only? I will consider moving this there.

Reply:
IMO this is only for testing which is not useful for users. this can be done in the same way as https://github.com/ceph/ceph-csi/blob/devel/e2e/utils.go#L261-L303 having a helper function to return the deployment is useful.

Reply:
I understand your point Madhu. What I'm saying is let's just keep the deployment in YAML form for readability. Will it be a concern if I move these templates to e2e/rbd/templates/ ?

Thanks in adavance!

Reply:
am fine with it as long as it's not for users. lets hear from others also